packages/terra-dropdown-button/src/_DropdownButtonBase.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Dropdown from './_Dropdown';
+import styles from './_DropdownButtonBase.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The button or buttons that are the primary option and caret
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The options to display in the dropdown. Should be of type `Button`.
+   */
+  buttons: PropTypes.node.isRequired,
```

Comment:
Change to items to match the file name changes.

Reply:
Fixed in 7b2111474afa8a329a9bc4035196cbf6f1264d25